metals/src/main/scala/scala/meta/internal/worksheets/WorkspaceEditWorksheetPublisher.scala

```diff
@@ -44,11 +46,10 @@ class WorkspaceEditWorksheetPublisher(buffers: Buffers)
         .toPosition(position)
       message <- getHoverMessage(snapshotPosition, messages.hovers)
     } yield new Hover(
-      List(
-        JEither.forRight[String, MarkedString](
-          new MarkedString("scala", message)
-        )
-      ).asJava
+      new MarkupContent(
+        MarkupKind.MARKDOWN,
+        HoverMarkup(message)
```

Comment:
I don't remember where exactly but when you did similar change you wrapped whole body in ```scala block, shouldn't it be similar here?

Reply:
I ended up making that happen in the apply method of `HoverMarkup`, so it is happening here :)

Reply:
perfect! 🥇